GitGuide.md

```diff
@@ -0,0 +1,45 @@
+# Git Guide 
+## Prerequisite
+1. At the heart of GitHub is an open source version control system (VCS) called Git. Git is responsible for everything GitHub-related that happens locally on your computer. 
+   1. [Download and install the latest version of GIT](https://git-scm.com/downloads)
+   2. [Set your username in GIT](https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git)
+   3. [Set your commit email address in Git](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)
+   4. [Authenticating with GitHub from Git](https://docs.github.com/en/get-started/quickstart/set-up-git#authenticating-with-github-from-git)
+   5. [Add SSH keys in GITHUB](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/about-ssh)
+2. It good to have some git desktop client to work fast. it is optional and totally depends upon the individual. Some recommendation are list down

```

Comment:
Some corrections to grammar:
1) It is* good to have
2) Some recommendations* are listed* down